ansible/index.md

```diff
@@ -1,21 +1,77 @@
 ---
 layout: page
 title: DE Ansible Documentation
-root: ../
 ---
 
-## Using this Repository
+# Table of Contents
+* [Overview](#overview)
+* [Setup](#setup)
+* [Design](#design)
+    * [Directory Layout](#directory-layout)
+    * [Role-Separated Playbooks](#role-separated-playbooks)
+    * [Inventories](#inventories)
+* [Playbooks](#playbooks)
 
-This repository is organized with group_vars defined with some default values in
-`inventories/group_vars/all`, with the intention that these playbooks are used with inventory 
-created in this `inventories` directory (use a `.cfg` extension to prevent committing the files) and
-group_vars files containing private and overriding values in a top-level `group_vars` directory.
+# Overview
+[Ansible](http://www.ansible.com/) is an open source, agentless automation tool. The DE development
+team uses Ansible to provision/update our servers and deploy the DE. However, for this repository,
+we only expose the Ansible scripts that we use for deploying the DE. 
 
-* [Setup](setup)
-* [Work Instructions](work_instructions)
-* [Builds](builds)
-* [Deployment Processes](deployments)
-* [ELK](elk)
-* [Inventory Requirements](inventory_requirements.html)
-* [Adhoc Commands](adhoc_commands.html)
+If you intend to use our ansible scripts, we highly suggest that you read the 
+[Ansible documentation](http://docs.ansible.com/ansible/index.html).
+
+# Setup
+
+* [Ansible Setup](setup/ansible.html)
+* [Docker-based Ansible Setup](setup/docker.html)
+
+# Design
+We have strived to follow Ansible's 
+[best practices](http://docs.ansible.com/ansible/playbooks_best_practices.html). 
+However, we have slightly diverged on the topics of 
+[directory layout](http://docs.ansible.com/ansible/playbooks_best_practices.html#directory-layout) and
+[role-separated top level playbooks](http://docs.ansible.com/ansible/playbooks_best_practices.html#top-level-playbooks-are-separated-by-role).
+
+## Directory Layout
+The ansible best practices for directory layout suggests using a `group_vars` folder, but you may 
+have noticed that the repo doesn't contain a `ansible/group_vars` folder. Our `group_vars` folder 
+resides in the `inventories` folder; 
+[ansible/inventories/group_vars]({{site.github.repository_url}}/tree/master/ansible/inventories/group_vars).
+Also, this folder contains a single file, `all`, which contains all of the variables used by the 
+provided roles and playbooks with default values.
+
+If you desire to override the defaults, you may edit the 
+[inventories/group_vars/all]({{site.github.repository_url}}/tree/master/ansible/group_vars/all) 
+file directly. Alternatively, you can create another file named `all`, but place it in a new 
+folder; `ansible/group_vars`. If you do this, the variables set in `ansible/group_vars/all` will 
+override the variables set in `ansible/inventories/group_vars/all`. You only need to include the 
+variables you wish to override.
+
```

Comment:
Note the original wording:
> organized with group_vars defined with some default values in
> `inventories/group_vars/all`, with the intention that these playbooks are used with ...
> group_vars files containing private and overriding values in a top-level `group_vars` directory.

I think we should remove the suggestion of editing the `ansible/inventories/group_vars/all` directly.